fastlane/lib/fastlane/helper/crashlytics_helper.rb

```diff
@@ -77,7 +71,6 @@ def generate_android_command(params)
           command << "-apiKey #{params[:api_token]}"
           command << "-apiSecret #{params[:build_secret]}"
           command << "-uploadDist #{File.expand_path(params[:apk_path]).shellescape}"
-          command << "-androidManifest #{xml.path.shellescape}"
```

Comment:
Same here. Keeping this here with the other parts of this commands is definitely preferred 👌